packages/core/test/client.test.ts

```diff
@@ -338,6 +328,80 @@ describe('client', function () {
+            at HoneybadgerTransport._write (/var/www/somebody/node_modules/winston-transport/index.js:82:19)
+            at doWrite (/var/www/somebody/node_modules/winston-transport/node_modules/readable-stream/lib/_stream_writable.js:409:139)
+            at writeOrBuffer (/var/www/somebody/node_modules/winston-transport/node_modules/readable-stream/lib/_stream_writable.js:398:5)
+            at HoneybadgerTransport.Writable.write (/var/www/somebody/node_modules/winston-transport/node_modules/readable-stream/lib/_stream_writable.js:307:11)
+            at DerivedLogger.ondata (/var/www/somebody/node_modules/winston/node_modules/readable-stream/lib/_stream_readable.js:681:20)
+            at DerivedLogger.emit (node:events:525:35)
+            at DerivedLogger.emit (node:domain:489:12)
+            at addChunk (/var/www/somebody/node_modules/winston/node_modules/readable-stream/lib/_stream_readable.js:298:12)
+            at readableAddChunk (/var/www/somebody/node_modules/winston/node_modules/readable-stream/lib/_stream_readable.js:280:11)
+            at DerivedLogger.Readable.push (/var/www/somebody/node_modules/winston/node_modules/readable-stream/lib/_stream_readable.js:241:10)
+            at DerivedLogger.Transform.push (/var/www/somebody/node_modules/winston/node_modules/readable-stream/lib/_stream_transform.js:139:32)
+            at DerivedLogger._transform (/var/www/somebody/node_modules/winston/lib/winston/logger.js:313:12)
+            at DerivedLogger.Transform._read (/var/www/somebody/node_modules/winston/node_modules/readable-stream/lib/_stream_transform.js:177:10)
+            at DerivedLogger.Transform._write (/var/www/somebody/node_modules/winston/node_modules/readable-stream/lib/_stream_transform.js:164:83)
+            at doWrite (/var/www/somebody/node_modules/winston/node_modules/readable-stream/lib/_stream_writable.js:409:139)
+            at writeOrBuffer (/var/www/somebody/node_modules/winston/node_modules/readable-stream/lib/_stream_writable.js:398:5)
+            at DerivedLogger.Writable.write (/var/www/somebody/node_modules/winston/node_modules/readable-stream/lib/_stream_writable.js:307:11)
+            at DerivedLogger.log (/var/www/somebody/node_modules/winston/lib/winston/logger.js:252:14)
+            at DerivedLogger.<computed> [as error] (/var/www/somebody/node_modules/winston/lib/winston/create-logger.js:95:19)
+            at console.hideMe [as error] (/var/www/somebody/node_modules/@somebody/logger/index.js:83:45)
+            at Function.logerror (/var/www/somebody/node_modules/express/lib/application.js:647:43)`
+      const backtrace = makeBacktrace(error.stack, true)
+      expect(backtrace[0]).toEqual({
+        file: '/var/www/somebody/node_modules/@somebody/logger/HoneybadgerTransport.js',
+        method: 'HoneybadgerTransport.log',
+        number: 18,
+        column: 19
+      })
+    })
+
+    it('filters out top frames that come from @honeybadger-io (browser)', function () {
```

Comment:
Might be helpful to also add a test where your code fails to find the honeybadger entries and just cuts the default number, mostly as a way to document the behavior.